packages/sewing-kit-koa/src/assets.ts

```diff
@@ -0,0 +1,88 @@
+import {join} from 'path';
+import {readJson} from 'fs-extra';
+import appRoot from 'app-root-path';
+
+export interface Asset {
+  path: string;
+  integrity?: string;
+}
+
+interface Entrypoint {
+  js: Asset[];
+  css: Asset[];
+}
+
+interface AssetList {
+  entrypoints: {[key: string]: Entrypoint};
+}
+
+interface Options {
+  assetHost: string;
+}
+
+export default class Assets {
+  assetHost: string;
+  private resolvedAssetList?: AssetList;
+
+  constructor({assetHost}: Options) {
+    this.assetHost = assetHost;
+  }
+
+  async scripts({name = 'main'} = {}) {
+    const {js} = getAssetsForEntrypoint(name, await this.getAssetList());
+
+    const scripts =
+      // Sewing Kit does not currently include the vendor DLL in its asset
+      // manifest, so we manually add it here (it only exists in dev).
+      // eslint-disable-next-line no-process-env
+      process.env.NODE_ENV === 'development'
+        ? [{path: `${this.assetHost}dll/vendor.js`}, ...js]
```

Comment:
I don't like that we are just hardcoding all of these. If sewing-kit ever changes we have nothing that will let us know this is broken other than just knowing.